lib/core/engine/iteration.js

```diff
@@ -195,6 +198,21 @@ class Iteration {
               result[i].browserScripts.pageinfo.visualElements
             );
             result[i].visualMetrics = videoMetrics.visualMetrics;
+
+            const profileFilename = path.join(
```

Comment:
Guard this with `browser === 'firefox'` whatever the flag for capturing Gecko profiles is, please.